registry/remote/repository.go

```diff
@@ -987,7 +1003,7 @@ func (s *manifestStore) generateDescriptor(resp *http.Response, ref registry.Ref
 	if serverHeaderDigestStr := resp.Header.Get(dockerContentDigestHeader); serverHeaderDigestStr != "" {
 		if serverHeaderDigest, err = digest.Parse(serverHeaderDigestStr); err != nil {
 			return ocispec.Descriptor{}, fmt.Errorf(
-				"%s %q: invalid response header value `%s`: `%s`; %w",
+				"%s %q: invalid response header value %s=%q; %w",
```

Comment:
`%s: %s` is the original form in the HTTP protocol.
```suggestion
				"%s %q: invalid response header value `%s: %s`; %w",
```

Reply:
Fixed.